contents/docs/llm-analytics/troubleshooting.mdx

```diff
@@ -0,0 +1,19 @@
+---
+title: Troubleshooting and FAQs
+---
+
+
+## How much does LLM analytics cost?
+
+Your first 100,000 `$ai_model` events each month are free – i.e. if you never exceed this number, you can use LLM analytics for free.
```

Comment:
I realize this is what pricing might be when we launch, but everywhere else in this PR it just says it is free, so if we ship this as is, it will be inconsistent until we update it